telnyx_rtc/build.gradle

```diff
@@ -123,6 +123,15 @@ android {
     testOptions {
         unitTests.includeAndroidResources = true
         unitTests.returnDefaultValues = true
+        testOptions {
```

Comment:
does this work? shouldn't be on the outer structure?
i.e:
```
testOptions {
        unitTests.includeAndroidResources = true
        unitTests.returnDefaultValues = true
        unitTests.all {
            if (name == "testDebugUnitTest" || name == "testReleaseUnitTest") {
                kover {
                    excludes = ["com\\.telnyx\\.webrtc\\.sdk\\.peer\\..*"]
                    //Can't mock native code, these methods will be tested via instrumentation instead.
                }
            }
        }
    }
```

Reply:
It actually works as is, which is weird. I'll have to double-check the syntax but it is working like it is at the moment so I don't think it matters. I will remove it if it isn't necessary though